sample-app/redisearch-node-rest/server.js

```diff
@@ -1,22 +1,19 @@
-const express = require('express');
-const cors = require('cors');
-const app = express();
-const bodyParser = require('body-parser');
-const serverPort = process.env.SERVER_PORT || 8086;
-
+import express from 'express';
+import cors from 'cors';
+import SearchService from './NodeSearchService.js';
 
+const app = express();
+app.use(cors());
+app.use(express.urlencoded({ extended: true }));
+app.use(express.json());
 
-const SearchService = require('./NodeSearchService');
+const serverPort = process.env.SERVER_PORT || 8086;
 const searchService = new SearchService();
 
-app.use(bodyParser.urlencoded({ extended: true }));
-app.use(cors());
-app.use(bodyParser.json());
-
-app.get('/api/1.0/movies/search', (req, res) => {
+app.get('/api/1.0/movies/search', async (req, res) => {
   const queryString = req.query.q;
-  const offset = Number((req.query.offset)?req.query.offset:'0');
-  const limit = Number((req.query.limit)?req.query.limit:'10');
+  const offset = Number((req.query.offset) ? req.query.offset : 0);
```

Comment:
change to `req.query.offset ?? 0` (it was added in node 14.0.0 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing#browser_compatibility)

edit: and validate it's a positive number